src/features/canvass/utils/markerIcon.tsx

```diff
@@ -2,52 +2,65 @@ import { FC } from 'react';
 import { useTheme } from '@mui/styles';
 import { lighten } from '@mui/system';
 
-import { ProgressState } from './getVisitState';
+import { VisitStats } from './getVisitPercentage';
 
 interface MarkerIconProps {
-  dataToShow?: 'done' | 'visited';
-  state?: ProgressState;
+  percentage?: VisitStats;
   selected: boolean;
+  uniqueKey?: string;
 }
 
 const MarkerIcon: FC<MarkerIconProps> = ({
-  dataToShow = 'visited',
-  state = 'none',
+  uniqueKey,
+  percentage,
   selected,
 }) => {
   const theme = useTheme();
-
-  const circleColors: Record<ProgressState, string> = {
-    all:
-      dataToShow == 'visited'
-        ? theme.palette.primary.main
-        : theme.palette.success.main,
-    none: theme.palette.grey[400],
-    some:
-      dataToShow == 'visited'
-        ? lighten(theme.palette.primary.main, 0.5)
-        : lighten(theme.palette.success.main, 0.5),
-  };
+  const totalVisitsKey = uniqueKey + '_totalVisits';
 
   return (
     <svg
-      fill="none"
+      fill="white"
       height="30"
+      style={{ filter: 'drop-shadow(0px 4px 2px rgba(0, 0, 0, 0.5))' }}
       viewBox="0 0 21 30"
       width="21"
       xmlns="http://www.w3.org/2000/svg"
     >
       <path
         d="M10.5 0C4.695 0 0 4.695 0 10.5C0 18.375 10.5 30 10.5 30C10.5 30 21 18.375 21 10.5C21 4.695 16.305 0 10.5 0Z"
-        fill={selected ? '#ED1C55' : '#848484'}
+        fill={selected ? '#ED1C55' : 'white'}
       />
-      <circle className="hole" cx="10.5" cy="10.5" fill="white" r="8.5" />
-      <circle
-        className="state-dot"
-        cx="10.5"
-        cy="10.5"
-        fill={circleColors[state]}
-        r="6.5"
+
```

Comment:
I think there should be a separate path for the white part of the progress indicator. Right now, the "white" part is not really a visual element at all, but instead it's the white background. So when a marker is selected, and the color of the background changes, it also affects the white part of the progress, which IMO is confusing.

|Default state|Selected state|
|-|-|
| ![image](https://github.com/user-attachments/assets/147309f3-653d-4e0e-83b0-b0e26b153015) | ![image](https://github.com/user-attachments/assets/90e0915f-6e85-4ec0-a461-e814434a6de8) |